types/postcss-inline-svg/package.json

```
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "postcss": "^8.3.0"
```

```suggestion
        "postcss": "^8.1.4"
```
https://github.com/TrySound/postcss-inline-svg/blob/master/package.json#L44